src/java/picard/analysis/artifacts/CollectSequencingArtifactMetrics.java

```diff
@@ -196,6 +199,18 @@ protected void acceptRead(final SAMRecord rec, final ReferenceSequence ref) {
             throw new PicardException("Record contains library that is missing from header: " + library);
         }
 
+        // set up some constants that don't change in the loop below
+        final int contextFullLength = 2 * CONTEXT_SIZE + 1;
+        final ArtifactCounter counter = artifactCounters.get(library);
+        final byte[] readBases = rec.getReadBases();
+        final byte[] readQuals;
+        if (USE_OQ) {
+            final byte[] tmp = rec.getOriginalBaseQualities();
+            readQuals = tmp == null ? rec.getBaseQualities() : tmp;
+        } else {
+            readQuals = rec.getOriginalBaseQualities();
```

Comment:
( this was actually @tfenne typo...that you copied...) should be rec.getBaseQualities() here.